posthog/hogql_queries/web_analytics/core_web_vitals_path_breakdown.py

```diff
@@ -0,0 +1,146 @@
+from posthog.hogql import ast
+from posthog.hogql.parser import (
+    parse_select,
+    parse_expr,
+)
+
+from posthog.hogql.query import execute_hogql_query
+
+from posthog.hogql_queries.web_analytics.web_analytics_query_runner import WebAnalyticsQueryRunner
+from posthog.schema import (
+    CachedWebStatsTableQueryResponse,
+    CoreWebVitalsPathBreakdownQuery,
+    CoreWebVitalsPathBreakdownQueryResponse,
+    CoreWebVitalsPathBreakdownResult,
+    CoreWebVitalsPathBreakdownResultItem,
+    CoreWebVitalsMetricBand,
+)
+from posthog.hogql.property import (
+    property_to_expr,
+    get_property_type,
+)
+from posthog.queries.trends.util import PROPERTY_MATH_FUNCTIONS
+
+
+class CoreWebVitalsPathBreakdownQueryRunner(WebAnalyticsQueryRunner):
+    query: CoreWebVitalsPathBreakdownQuery
+    response: CoreWebVitalsPathBreakdownQueryResponse
+    cached_response: CachedWebStatsTableQueryResponse
+
+    def to_query(self):
+        return parse_select(
+            "SELECT * FROM {good_query} UNION ALL {needs_improvements_query} UNION ALL {poor_query}",
```

Comment:
Not 100% sure, but I don't think this is going to be optimised by clickhouse at all. I think it'll do something like 3 queries and stitch the results together. There might be a more performant way of structuring this.

(don't let this stop you from merging, it's fine to make it work, and make it fast, in 2 separate steps)

Reply:
Yeah, I'm pretty sure that's 3 queries, unfortunately. I've tried a new approach in 9852a28523df02ea194d6dba721ef04969acaadf